tests/ec2client_test.py

```diff
@@ -48,6 +49,43 @@ def test_create_instance(self):
                                           'VolumeType': 'gp2'}}]
         )
 
+        ec2client.create_instance(
```

Comment:
You could add a comment here describing what your test does, and reset the ec2_conn.run_instances.call_count so as to make the assertEqual() below properly show a single call to EC2. 
ec2_conn.reset_mock()